launch_ros/launch_ros/utilities/to_parameters_list.py

```diff
@@ -75,7 +75,7 @@ def to_parameters_list(
     """
     parameters = []  # type: List[rclpy.parameter.Parameter]
     node_name = node_name.lstrip('/')
-    if namespace:
+    if namespace and namespace != '/':
```

Comment:
This handles an edge case where `namespace == '/'`, which would result in a node name with a forward slash, e.g. `/test_node_name`. But, the rest of this function expects the slashes to be stripped.